### DISCUSSION TITLE
Replace "upcoming" with a countdown clock to the start time

### DISCUSSION THREAD

**Discussion by Deanbau** [f1multiviewer/issue-tracker#567] [Feature requests]
As the title.. would be good to see a countdown to see when the stream starts (22:00 in this instance) 

<img width="936" alt="image" src="https://github.com/f1multiviewer/issue-tracker/assets/39572328/61711a6f-3e80-43f4-a407-23aaab65b7a9">

**Comment by jeppeolesen** [comment]
Redirecting this to #137
